packages/arb-token-bridge-ui/src/components/TransferPanel/hooks/useRouteStore.ts

```diff
@@ -0,0 +1,22 @@
+import { create } from 'zustand'
+
+export type RouteType = 'arbitrum' | 'layerzero' | 'cctp'
```

Comment:
I think this should be an enum, and this should come from `bridge-sdk` and should be linked with transfer-type otherwise we're creating a lot of disjointed types trying to express the same information?

Eg. in bridge-sdk, we have `TransferType = `${Asset}_${TxType}` | 'cctp' | 'oftV2'`. 
At the very least the Route names and Transfer type names should be consistent.

Reply:
TransferType carries a lot of information that we don't need, we have only 3 types here. And TransferType has 8. Also in ArbitrumRoute for example we would have a transferType that might be different than the actual transfer